src/components/ec-filter-popover/ec-filter-popover.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div
+    class="ec-filter-popover"
+    data-test="ec-filter-popover"
+  >
+    <ec-popover
+      popover-class="ec-popover"
+      placement="bottom"
+      trigger="click"
+      level="modal"
+      @update:open="clickTrigger"
+    >
+      <div class="ec-filter-popover__filter-item">
+        <a class="ec-filter-popover__label">{{ label }}</a>
+        <!-- <ec-badge-number size="20" value="numberOfSelectedFilters" />
+        TODO Badge with https://fxsolutions.atlassian.net/browse/ONL-4909 -->
+        <ec-icon
+          class="ec-filter-popover__icon"
+          :size="16"
+          name="simple-chevron-down"
+          type="interactive"
+        />
+      </div>
+
+      <div
+        slot="popover"
+        class="ec-filter-popover__filter-content"
+      >
+        <slot
+          name="filter"
+        />
+      </div>
+    </ec-popover>
+  </div>
+</template>
+<script>
+import EcPopover from '../ec-popover';
+import EcIcon from '../ec-icon';
+
+export default {
+  name: 'EcFilterPopover',
+  components: { EcPopover, EcIcon },
+  props: {
+    label: {
+      type: String,
+      required: true,
+      default: '',
+    },
+    numberOfSelectedFilters: {
+      type: Number,
+      required: false,
+      default: 0,
+    },
+  },
+  data() {
+    return {
+      isOpen: false,
+    };
+  },
+  methods: {
+    clickTrigger(status) {
+      this.isOpen = status;
+      this.$emit('isOpen', this.isOpen);
```

Comment:
You can emit status directly, the name of the event should be just `open`, so we can use it in the parent as
```
<ec-filter-popover @open="doSomething()">
```
instead of
```
<ec-filter-popover @is-open="doSomething()">
```

Reply:
I've removed the emit for now